pkg/transport/bprotocol/constants.go

```diff
@@ -6,7 +6,7 @@ const (
 	BidAcceptedProtocolID = "/bacalhau/compute/bid_accepted/1.0.0"
 	BidRejectedProtocolID = "/bacalhau/compute/bid_rejected/1.0.0"
 	CancelProtocolID      = "/bacalhau/compute/cancel/1.0.0"
-	ExecutionLogsID       = "/bacalhau/compute/executionlogs/1.0.0"
+	ExecutionLogsID       = "/bacalhau/compute/execution_logs/1.0.0"
```

Comment:
I think this is going to break compatibility with all previous version of this, any reason to change this?

Reply:
yeah we have a new protocol for log vending between the compute node and the requester. Meaning we now need both compute and requester nodes to be on the same version for logging to work. We can utilize `NewMinVersionNodeRanker` to make sure requester won't communicate with lower version compute nodes, but I've checked our graphana metrics for how often users call our current logging API, and I barely saw 4 calls the past month. 

This should be a concern since it is not an actively used API, and just a client update is needed

Reply:
> yeah we have a new protocol for log vending between the compute node and the requester. 

Could we just bump the protocol version then?